pytest.ini

```diff
@@ -34,9 +37,8 @@ testpaths =
     tests/setops_test.py
     tests/sort_test.py
     tests/string_test.py
+    tests/util_test.py
```

Comment:
it seems we're still missing `summarization_test.py` which should come just before this one